aztec/src/main/kotlin/org/wordpress/aztec/toolbar/AztecToolbar.kt

```diff
@@ -738,6 +740,10 @@ class AztecToolbar : FrameLayout, OnMenuItemClickListener {
     }
 
     private fun setupMediaToolbar() {
+        val mediaToolbarContainer : LinearLayout = findViewById(R.id.media_button_container)
```

Comment:
Can we make this `LinearLayout` a more generic `View` instead? Just in case we want to change it in the future, and as all we really care about for now is just to be able to make the view `VISIBLE` or `GONE` only